### DISCUSSION TITLE
2024/05/23/tcp-nutssb-%e6%96%b9%e5%bc%8f%e6%89%93%e6%b4%9e%e5%a4%87%e5%bf%98/

### DISCUSSION THREAD

**Discussion by giscus** [snltty/linker#15] [Announcements]
# 2024/05/23/tcp-nutssb-%e6%96%b9%e5%bc%8f%e6%89%93%e6%b4%9e%e5%a4%87%e5%bf%98/

# Tcp Nutssb，基于低TTL的打洞备忘 - snltty

https://blog.snltty.com/2024/05/23/tcp-nutssb-%e6%96%b9%e5%bc%8f%e6%89%93%e6%b4%9e%e5%a4%87%e5%bf%98/

**Comment by ChiuSky** [comment]
博主，您好，我最近也有研究打洞，有碰到联通与移动的4G，NAT4网络无法打洞，但是电信的能成功，希望能请教一下
